.gitignore

```diff
@@ -35,3 +39,8 @@ test.jfr
 tlatools/org.lamport.tlatools/test-model/test.out
 .tlacache/
 .Rproj.user
+
+toolbox/**/.settings/*.prefs
```

Comment:
The .prefs and .project files are required by our Eclipse IDE, and, thus, should not be ignored.